django_rq_dashboard/views.py

```diff
@@ -78,8 +78,14 @@ def serialize_scheduled_queues(queue):
 
 class SuperUserMixin(object):
     def dispatch(self, request, *args, **kwargs):
-        if not request.user.is_superuser:
-            raise PermissionDenied
+        if not request.user.is_authenticated():
+            return redirect('{}?{}'.format(reverse('admin:login'),
+                                           urllib.urlencode({
```

Comment:
Thanks :) But then the path becomes `urllib.parse.urlencode` -- you can do `from django.utils.six.moves.urllib.parse import urlencode`